docs: serialization clarification (#45480)
metabase/metabase: docs/installation-and-operation/serialization.md

- **Staging environments**. Enable a staging-to-production workflow for important dashboards by exporting from a staging instance of Metabase and then importing them into your production instance(s).
- **Version control**. Check the exported files into version control and audit changes to them, as the YAML files contained within the export are pretty readable.

> - **Duplicating assets between Metabase instances**. Export the "template" data from a source Metabase instance and import them to one or more target instances to duplicate.

Reviewer: This seems like a restatement of the first bullet, but I'm not opposed to including it if it helps people get the point.

And should it be "between" or "to"? Does "between" imply bi-directionality?

```suggestion
- **Duplicating assets to other Metabase instances**. Export the "template" data from a source Metabase and import them to one or more target instances.
```

Check out our guides for:

- [Running multiple environments](https://www.metabase.com/learn/administration/multi-env)
- [Setting up git-based workflow](https://www.metabase.com/learn/administration/git-based-workflow)

> > Serialization isn't intended for use cases like duplicating assets or swapping data sources _within the same Metabase instance_. If you're using serialization for duplicating assets within the same instance, check out [How export works](#how-export-works), [How import works](#how-import-works), and the directions for your use case in [Other uses of serialization](#other-uses-of-serialization)

Reviewer: We should emphasize "within", as that's the key distinction:

```suggestion
> Serialization isn't intended for use cases like duplicating assets or swapping data sources _within_ the same Metabase instance. If you're using serialization for duplicating assets within the same instance, check out [How export works](#how-export-works), [How import works](#how-import-works), and the directions for your use case in [Other uses of serialization](#other-uses-of-serialization)
```

## Other uses of serialization

> Serialization is intended for version control, staging-to-production workflows, and duplicating assets between Metabase instances. While it is possible to use serialization for other use cases like duplicating assets within a single instance, these use cases are not currently officially supported.

Reviewer: ```suggestion
Serialization is intended for version control, staging-to-production workflows, and duplicating assets to other Metabase instances. While it's possible to use serialization for other use cases (like duplicating assets _within_ a single instance), we don't officially support these use cases.
```

> We're providing some directions on how to approach these alternative use cases, but you should use them at your own risk. We strongly recommend that you test any process involving serialization on a non-production instance first, and reach out to [help@metabase.com](mailto:help@metabase.com) if you have any questions.

Reviewer: "Unsupported" is more precise than "alternative".

```suggestion
We're providing some directions on how to approach these unsupported use cases, but you should use them at your own risk. We strongly recommend that you test any process involving serialization on a non-production instance first, and reach out to [help@metabase.com](mailto:help@metabase.com) if you have any questions.
```
